### DISCUSSION TITLE
Can't login to Gimp bug reports with Github login

### DISCUSSION THREAD

**Discussion by lohikarhu** [community/community#48536] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Trying to report a Gimp bug, tried to use Github login, as shown at bottom of Sign In, and it just dumps me back to the same Gnome Gitlab sign in page

It also appears to have started taking my new registration, so has set aside the Username, but I can't finish registration

All because Gimp 2.99.14 crashes on Mojave  (yeah, it's old, but runs...)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @lohikarhu ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
